docs/source/models/supported_models.rst

```diff
@@ -155,10 +155,6 @@ Decoder-only Language Models
     - Persimmon
     - :code:`adept/persimmon-8b-base`, :code:`adept/persimmon-8b-chat`, etc.
     - 
-  * - :code:`QWenLMHeadModel`
-    - Qwen
-    - :code:`Qwen/Qwen-7B`, :code:`Qwen/Qwen-7B-Chat`, etc.
-    -
```

Comment:
Since the model can also be used in text-only mode, we should still keep it here (so it appears in two tables).

Reply:
Added it back!